Lib/pathlib.py

```diff
@@ -14,15 +14,6 @@
 
 
 supports_symlinks = True
```

Comment:
This is left for backward compatibility in 3.9, but it's always True because the only way for it to get False is a pre-Vista version of Windows. Python 3.9 only supports Windows 8.1+.